doc/builders/packages/fish.section.md

```diff
@@ -0,0 +1,49 @@
+# Fish {#sec-fish}
+
+Fish is a "smart and user-friendly command line shell" with plugins.
+
+
+## Vendor Fish scripts {#sec-fish-vendor}
+
+Any package may ship its own Fish completions, configuration snippets, and
+functions. Those should be installed to
+`$out/share/fish/vendor_{completions,conf,functions}.d` respectively.
+
+When the `programs.fish.enable` and
+`programs.fish.vendor.{completions,config,functions}.enable` options from the
```

Comment:
Do we want to make this the same as the folder name (e.g. `programs.fish.vendor.conf.enable` rather than `programs.fish.vendor.config.enable`)?

Reply:
Both keywords are used in the Fish documentation.
I'd say that `config` is better for discoverability when searching on the option webpage.

Reply:
Sounds reasonable to me.